features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/admin-dashboard/site/themes/default/templates/menu/left/template.jag

```diff
@@ -70,6 +70,25 @@
             </ul>
         </li>
 
+        <li class="dropdown custom-verticle">
+            <a href="<%=listMappedUrl%>?task=themeUpload" class="dropdown-toggle" data-toggle="dropdown">
+            <i class="icon-user"></i> <br> <%= i18n.localize("customTheme") %>  <i class="icon-chevron-right"></i>
```

Comment:
I think we can use icon-eye-open for this among the icon sets we have in http://getbootstrap.com/2.3.2/base-css.html#icons